org-code-javabuilder/playground/src/main/java/org/code/playground/TextItem.java

```diff
@@ -0,0 +1,140 @@
+ public class TextItem implements Item {
+    /**
+     * Creates text item that can be placed the board.
+     *
+     * @param text the text content
+     * @param x the distance from the left side of the board
+     * @param y the distance from the top of the board
+     * @param color the color to draw the text
+     * @param font the name of the font to draw the text
+     * @param fontStyle the style of the font
+     * @param height the height of the text in pixels
+     * @param rotation the rotation or tilt of the text, in degrees
+     */
+    public TextItem(String text, Color color, Font font, FontStyle fontStyle, int height, double rotation);
```

Comment:
Constructor should also take x, y, width, and height right?

Reply:
x,y and yes, height yes, but not width as text width varies based on the length of the string.